dashboard/test/controllers/api/v1/pd/regional_partner_mini_contacts_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class Api::V1::Pd::RegionalPartnerMiniContactsControllerTest < ActionDispatch::IntegrationTest
+  test 'can create a new regional partner mini contact' do
+    # Use the same state & zip as the mini-contact factory's defaults.
+    state = 'OH'
+    zip = '45242'
+
+    regional_partner = create :regional_partner, name: "partner_OH_45242"
+    regional_partner.mappings.find_or_create_by!(state: state)
+    regional_partner.mappings.find_or_create_by!(zip_code: zip)
+
+    assert_valid_form build(:pd_regional_partner_mini_contact_hash)
+  end
+
+  test 'create returns error if email is missing' do
+    form_data = build(:pd_regional_partner_mini_contact_hash).
+      merge("email" => "")
```

Comment:
Surprised you can't
```rb
form_data = build(:pd_regional_partner_mini_contact_hash, email: "")
```

Reply:
I inherited the technique from the existing tests, but I think it doesn't work because we're working with a `Hash` here: ```NoMethodError: undefined method `email=' for {}:Hash```

Reply:
Oh right - because the method is `:[]=`, not `email=`.